common/changes/office-ui-fabric-react/elliberm-gototoday_2018-07-11-19-51.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "don't select date by defaultick on Today",
```

Comment:
Before you check in, could you please fix this to be a proper changelog entry? (This gets recorded/published into the release notes https://github.com/OfficeDev/office-ui-fabric-react/releases)

Something like: Calendar: [......]